.gitignore

```diff
@@ -2,6 +2,7 @@
 .idea
 .vscode
 .protractor-fail-fast
+*.DS_Store
```

Comment:
I think this should just be...

```suggestion
.DS_Store
```

(Sorry for missing that on my first review.)